tests/src/Functional/ContentPublicationRenderTest.php

```diff
@@ -496,4 +496,87 @@ public function testPublicationRendering(): void {
     $field_list_assert->assertPattern($details_expected_values, $content_items[0]->getHtml());
   }
 
+  /**
+   * Tests that the Publication thumbnail media renders the translated media.
+   */
+  public function testPublicationThumbnailMediaTranslation(): void {
+    // Make publication node and image media translatable.
+    \Drupal::service('content_translation.manager')->setEnabled('node', 'oe_publication', TRUE);
```

Comment:
The bundle is already translatable because this is done in the `setUp()` method.